src/feedback/vue/submitFeedback.vue

```diff
@@ -70,22 +70,33 @@ export default defineComponent({
             sentiment: '',
             isSubmitting: false,
             error: '',
+            feedbackName: '',
         }
     },
+    created() {
+        this.getName()
+    },
     methods: {
         async submitFeedback() {
             this.error = ''
             this.isSubmitting = true
             console.log('Submitting feedback...')
-
+            // identifier to help us (internally) know that feedback came from either CodeWhisperer or AWS Toolkit
             const resp = await client.submit({
-                comment: this.comment,
+                comment:
+                    this.feedbackName === 'CodeWhisperer' ? 'CodeWhisperer onboarding: ' + this.comment : this.comment,
```

Comment:
is it really only for onboarding? would it make sense to add a "Send Feedback" item in the right-click menu of the CodeWhisperer node (in "developer tools" panel)?

Reply:
This additional 'CodeWhisperer onboarding: ' was added to separate Feedback of CodeWhisperer from feedback of AWS Toolkit.

Reply:
Yes. So would it potentially be used for other kinds of codewhisperer feeding besides "onboarding"? If so, the `comment` could be set to "CodeWhisperer". No need to mention "onboarding" in the `comment`.

Reply:
It potentially can have other source from CodeWhisperer e.g. "CodeWhisperer suggestions" in the future, but not as of now.